pythonforandroid/bootstrap.py

```diff
@@ -271,6 +272,15 @@ def fry_eggs(self, sitepackages):
                     shprint(sh.mv, '-t', sitepackages, *files)
                 shprint(sh.rm, '-rf', d)
 
+    def apk(self, build_mode, env):
```

Comment:
Does this function actually get called from somewhere? I see the ant/gradle build stuff is handled elsewhere.